package.json

```diff
@@ -1,31 +1,27 @@
 {
-  "name": "@openfeature/js-sdk",
-  "version": "1.1.0",
+  "name": "@openfeature/js",
+  "version": "0.0.0",
```

Comment:
Is the version used?

Reply:
I don't think this version is used for anything, no. The root project is not published.

Reply:
It may be worth explicitly setting this package.json to private.
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#private

Reply:
You might be right about that, I think it makes sense.

Reply:
Done.